db/migrate.go

```diff
@@ -0,0 +1,30 @@
+package db
+
+import (
+	"database/sql"
+	"embed"
+
+	"github.com/pressly/goose/v3"
+)
+
+//go:embed migrations/*.sql
+var embedMigrations embed.FS
+
+func Migrate(db *sql.DB) error {
+	goose.SetBaseFS(embedMigrations)
+
+	if err := goose.SetDialect("sqlite3"); err != nil {
+		return err
+	}
+
+	err := goose.Version(db, "migrations")
```

Comment:
Unfortunately this API call just prints out the version via the default `log`, rather than returning it :/